.setup/distro_setup/debian/setup_distro.sh

```diff
@@ -128,7 +128,7 @@ hardening-includes p7zip-full patchutils \
 libpq-dev unzip valgrind zip libmagic-ocaml-dev common-lisp-controller libboost-all-dev \
 javascript-common  \
 libfile-mmagic-perl libgnupg-interface-perl libbsd-resource-perl libarchive-zip-perl gcc g++ \
-g++-multilib jq flex bison spim poppler-utils pdftk
+g++-multilib jq flex bison spim poppler-utils PyPDF2
```

Comment:
Move this to `.setup/install_system.sh` and install it via pip3. It's easier for us to keep up-to-date on python packages using pip than apt.